config/parse-overrides/src/index.ts

```diff
@@ -41,7 +41,7 @@ export function parseOverrides (
 function parsePkgSelector (selector: string) {
   const wantedDep = parseWantedDependency(selector)
   if (!wantedDep.alias) {
-    throw new PnpmError('INVALID_OVERRIDE_SELECTOR', `Cannot parse the "${selector}" selector in the overrides`)
+    throw new ParseSelectorError('INVALID_OVERRIDE_SELECTOR', 'pnpm.overrides', selector)
```

Comment:
This doesn't make sense to me. You specify that it is an error with `pnpm.overrides` although it can be an error with a different field as well. This function doesn't info about context. Probably just a try/catch to both cases when it is used.

Reply:
Could you clarify here? I added the selector to this error so that I wouldn’t have to parse that information out in the try/catch re-throw case. 

Are you suggesting that I remove the context field and have a general error message that works for both selectors? 

Generalizing it would allow me to remove the try/catch re-throw code. 

Or are you asking me to remove the “field” constructor parameter in favor of a “message”, and each throw constructs its own message?